pkg/ocm/helpers.go

```diff
@@ -493,16 +499,16 @@ func GetVersionMinor(ver string) string {
 	return fmt.Sprintf("%d.%d", segments[0], segments[1])
 }
 
-func CheckSupportedVersion(clusterVersion string, operatorVersion string) (bool, error) {
-	v1, err := semver.NewVersion(clusterVersion)
+func IsGreaterThanOrEqual(version1, version2 string) (bool, error) {
+	versionPrefix := "openshift-v"
```

Comment:
I see we have a function `GetRawVersionId`, maybe we can use it before calling `IsGreaterThanOrEqual`.

Reply:
Good point, will use it, thanks.